hub/policy/src/integration-test/java/uk/gov/ida/integrationtest/hub/policy/rest/Cycle3DTO.java

```diff
@@ -91,4 +92,8 @@ public String getEncryptedMatchingDatasetAssertion() {
     public boolean getTransactionSupportsEidas() {
         return transactionSupportsEidas;
     }
+
+    public boolean isRegistering() {
```

Comment:
It looks like this is always going to be false.

a) is that right?
b) if so, can we just `return false;` and skip the private field?

Reply:
Fair point. And the same goes for transactionSupportsEidas and getTransactionSupportsEidas just above

Reply:
Actually, these cannot be removed as this is a DTO object used in integration tests and these fields are expected to be present by the JSON validator. Adding them back in.